docs/developers/README.md

```diff
@@ -133,23 +135,64 @@ If the image is a private registry, the service account should include an
 
 ## Reserved directories
 
-The `/tekton/` directory is reserved on containers for internal usage. Examples
-of how this directory is used:
-
-* `/workspace` - This directory is where [resources](#resources) and [workspaces](#workspaces)
-  are mounted.
-* `/tekton` - This directory is used for Tekton specific functionality:
-  * These folders are [part of the Tekton API](../api_compatibility_policy.md):
-    * `/tekton/results` is where [results](#results) are written to
-      (path available to `Task` authors via [`$(results.name.path)`](../variables.md))
-    * `/tekton/steps` is where the `step` exitCodes are written to
-      (path available to `Task` authors via [`$(steps.<stepName>.exitCode.path)`](../variables.md#variables-available-in-a-task))
-  * These folders are implementation details of Tekton and **users should not
-    rely on this specific behavior as it may change in the future**:
-    * `/tekton/tools` contains tools like the [entrypoint binary](#entrypoint-rewriting-and-step-ordering)
-    * `/tekton/termination` is where the eventual [termination log message](https://kubernetes.io/docs/tasks/debug-application-cluster/determine-reason-pod-failure/#writing-and-reading-a-termination-message) is written to
-    * [Sequencing step containers](#entrypoint-rewriting-and-step-ordering)
-      is done using both `/tekton/downward/ready` and numbered files in `/tekton/tools`
+### /workspace
+
+- `/workspace` - This directory is where [resources](#resources) and
+  [workspaces](#workspaces) are mounted.
+
+### /tekton
+
+The `/tekton/` directory is reserved on containers for internal usage.
+
+Here is an example of a directory layout for a simple Task with 2 script steps:
+
+```
+/tekton
+|-- creds
+|-- downward
+|   |-- ..2021_09_16_18_31_06.270542700
+|   |   `-- ready
+|   |-- ..data -> ..2021_09_16_18_31_06.270542700
+|   `-- ready -> ..data/ready
+|-- home
+|-- results
+|-- scripts
+|   |-- script-0-t4jd8
+|   `-- script-1-4pjwp
+|-- steps
+|   |-- 0 -> /tekton/steps/step-unnamed-0
+|   |-- 1 -> /tekton/steps/step-foo
+|   |-- step-foo
+|   `-- step-unnamed-0
+|       `-- exitCode
+|-- termination
+`-- tools
+    |-- 0
+    `-- entrypoint
```

Comment:
are these backticks part of the directory layout? https://github.com/tektoncd/pipeline/blob/e084563cfc926418b7211f9388857bbbf26e8bf2/docs/developers/README.md#tekton

@sbwsg has explained some of the layout and files in a video [here](https://youtu.be/-zVFIVjT36A). While we are revamping the developers doc, can we include a link to the video recording?

Reply:
> are these backticks part of the directory layout?

The backticks are the formatting of `tree` as the opposite of `/` to show that the end of the directory. My guess is was a style choice since `` `- `` matches the horizontal line height instead of `\-`

> @sbwsg has explained some of the layout and files in a video here. While we are revamping the developers doc, can we include a link to the video recording?

There's a few useful videos there, so added a link to the top of the page!